# XYZ company - mean sales generated by each zone

In [2]:
import pandas as pd
import seaborn as sns
import scipy

In [3]:
data=pd.read_csv('Sales_data_zone_wise.csv')
data.describe()

Zone - A      Zone - B      Zone - C      Zone - D
count  2.900000e+01  2.900000e+01  2.900000e+01  2.900000e+01
mean   1.540493e+06  1.755560e+06  1.772871e+06  1.842927e+06
std    2.619401e+05  1.683899e+05  3.331937e+05  3.750165e+05
min    1.128185e+06  1.527574e+06  1.237722e+06  1.234311e+06
25%    1.305972e+06  1.606010e+06  1.523308e+06  1.520406e+06
50%    1.534390e+06  1.740365e+06  1.767047e+06  1.854412e+06
75%    1.820196e+06  1.875658e+06  2.098463e+06  2.180416e+06
max    2.004480e+06  2.091194e+06  2.290580e+06  2.364132e+06

In [4]:
data.head()

Month  Zone - A  Zone - B  Zone - C  Zone - D
0  Month - 1   1483525   1748451   1523308   2267260
1  Month - 2   1238428   1707421   2212113   1994341
2  Month - 3   1860771   2091194   1282374   1241600
3  Month - 4   1871571   1759617   2290580   2252681
4  Month - 5   1244922   1606010   1818334   1326062

In [5]:
data.isnull().sum()

Month       0
Zone - A    0
Zone - B    0
Zone - C    0
Zone - D    0
dtype: int64

In [22]:
data.mean()

Zone - A    1.540493e+06
Zone - B    1.755560e+06
Zone - C    1.772871e+06
Zone - D    1.842927e+06
dtype: float64

**From the above, it is seen that Zone D has the highest sales mean value of 1.843 million $**

# Sum of sales by month

In [28]:
data['month_total']=data.sum(axis=1)
data.loc[:, ['Month','month_total']]

Month  month_total
0    Month - 1     28090176
1    Month - 2     28609212
2    Month - 3     25903756
3    Month - 4     32697796
4    Month - 5     23981312
5    Month - 6     28605548
6    Month - 7     29148432
7    Month - 8     31265196
8    Month - 9     26813580
9   Month - 10     28512840
10  Month - 11     28131132
11  Month - 12     24444336
12  Month - 13     23701696
13  Month - 14     28622060
14  Month - 15     23736624
15  Month - 16     26026636
16  Month - 17     28597532
17  Month - 18     28333960
18  Month - 19     27887812
19  Month - 20     28498396
20  Month - 21     29558388
21  Month - 22     30240004
22  Month - 23     26751676
23  Month - 24     31138988
24  Month - 25     24383672
25  Month - 26     26049440
26  Month - 27     25071672
27  Month - 28     29883680
28  Month - 29     27089108

# H0 - All zones have equal mean sales
# H1 - Atleast one zone has differing mean sales from others

In [29]:
from scipy import stats
from scipy.stats import f_oneway
f, p= scipy.stats.f_oneway(data['Zone - A'],data['Zone - B'], data['Zone - C'],data['Zone - D'])

In [30]:
print(f,p)

5.672056106843581 0.0011827601694503335


# Since p<0.05, we reject the null hypothesis. Therefore, the alternate hypothesis of atleast one zone having differing mean sales is upheld.